### PR TITLE
PEP8: fix flake8 errors

### PR DESCRIPTION
**Pull request by neteler**
This PR fixes PEP8 errors identified with flake8 and selectively fixed with autopep8.

For easier review, the different errors and warnings are separate out into single commits.

### CODE REVIEW

**Comment by neteler** [comment]
Leftover are:

- 4 x   F403 'from mdlib.cswutil import *' used; unable to detect undefined names
- 33 x F405 'open_url' may be undefined, or defined from star imports: .cswutil
- 1 x    F632 use ==/!= to compare constant literals (str, bytes, int, float, tuple)
- 2 x    F811 redefinition of unused 'CatalogueServiceWeb' from line 18

My test script:

```
cat ~/bin/qa_run_flake8.sh 
#!/bin/sh

# Markus Neteler, 2020
# script to recursively check GRASS GIS Python scripts for selected PEP8 errors

# Fedora: sudo pip3 install flake8 flake8-colors 
# Ubuntu: pip3 install flake8 flake8-colors

# fetch/update GRASS GIS optimized config
wget --quiet https://raw.githubusercontent.com/OSGeo/grass/master/scripts/.flake8 -O .flake8
echo 'format = ${cyan}%(path)s${reset}:${yellow_bold}%(row)d${reset}:${green_bold}%(col)d${reset}: ${red_bold}%(code)s${reset} %(text)s' >> .flake8

# run flake8 on directory tree
flake8 --count --statistics --show-source --jobs=4 .

rm -f .flake8
```

File: grass7/gui/wxpython/wx.metadata/g.gui.metadata/g.gui.metadata.py [hunk 1]
```diff
@@ -1182,7 +1182,7 @@ def initTemporalTree(self, location, mapset):
                         self.AppendItem(vartype, ml[0])
 
         except Exception as e:
-            GError('Initialize of temporal tree catalogue error: < %s >'%e)
+            GError('Initialize of temporal tree catalogue error: < %s >' %e)
```
Comment (tmszi) [inline grass7/gui/wxpython/wx.metadata/g.gui.metadata/g.gui.metadata.py:1185]:
```suggestion
            GError('Initialize of temporal tree catalogue error: < %s >' % e)
```

File: grass7/gui/wxpython/wx.metadata/mdlib/mdeditorfactory.py [hunk 2]
```diff
@@ -131,7 +131,7 @@ def saveToXML(self, md, owsTagList, jinjaPath, outPath=None, xmlOutName=None, ms
             io.write(str1)
             io.close()
         except Exception as err:
-            print("WARNING: Cannot check and remove non ascii characters from template err:< %s >"%err)
+            print("WARNING: Cannot check and remove non ascii characters from template err:< %s >" %err)
```
Comment (tmszi) [inline grass7/gui/wxpython/wx.metadata/mdlib/mdeditorfactory.py:134]:
```suggestion
            print("WARNING: Cannot check and remove non ascii characters from template err:< %s >" % err)
```

File: grass7/gui/wxpython/wx.metadata/mdlib/mdeditorfactory.py [hunk 3]
```diff
@@ -803,7 +804,7 @@ def __init__(self,parent,mdObject,mdOWS):
         self.comboKeysLabel=wx.StaticText(parent=self.parent,id=ID_ANY,label='Keywords from repositories')
         self.comboKeys=wx.ComboBox(parent=self.parent, id=ID_ANY)
 
-        self.keysList=wx.TreeCtrl(parent=self.parent, id=ID_ANY,size=(0, 120),style=wx.TR_FULL_ROW_HIGHLIGHT|wx.TR_DEFAULT_STYLE)
+        self.keysList=wx.TreeCtrl(parent=self.parent, id=ID_ANY,size=(0, 120),style=wx.TR_FULL_ROW_HIGHLIGHT |wx.TR_DEFAULT_STYLE)
```
Comment (tmszi) [inline grass7/gui/wxpython/wx.metadata/mdlib/mdeditorfactory.py:807]:
```suggestion
        self.keysList=wx.TreeCtrl(parent=self.parent, id=ID_ANY, size=(0, 120), style=wx.TR_FULL_ROW_HIGHLIGHT | wx.TR_DEFAULT_STYLE)
```

File: grass7/gui/wxpython/wx.metadata/mdlib/mdgrass.py [hunk 4]
```diff
@@ -219,13 +220,13 @@ def wkt2standards(self,prj_txt):
         try:
             from osgeo import osr
         except Exception as e:
-            grass.message('GDAL python library is not installed: %s \n identifying of EPSG is disabled'%e)
+            grass.message('GDAL python library is not installed: %s \n identifying of EPSG is disabled' %e)
```
Comment (tmszi) [inline grass7/gui/wxpython/wx.metadata/mdlib/mdgrass.py:223]:
```suggestion
            grass.message('GDAL python library is not installed: %s \n identifying of EPSG is disabled' % e)
```

File: grass7/gui/wxpython/wx.metadata/mdlib/mdgrass.py [hunk 5]
```diff
@@ -361,7 +362,7 @@ def createGrassBasicISO(self, profile=None):
         epsg=self.getEPSG()
         if epsg is not None:
             self.md.referencesystem=MD_ReferenceSystem(None)
-            self.md.referencesystem.code='http://www.opengis.net/def/crs/EPSG/0/%s'%epsg
+            self.md.referencesystem.code='http://www.opengis.net/def/crs/EPSG/0/%s' %epsg
```
Comment (tmszi) [inline grass7/gui/wxpython/wx.metadata/mdlib/mdgrass.py:365]:
```suggestion
            self.md.referencesystem.code = 'http://www.opengis.net/def/crs/EPSG/0/%s' % epsg
```

**Review by tmszi — CHANGES_REQUESTED** [review]

**Comment by neteler** [comment]
Thanks, @tmszi !

**Comment by neteler** [comment]
The open issues as per https://github.com/OSGeo/grass-addons/pull/290#issuecomment-699902744 might be addressed in separate PR(s).

If you agree, I'd merge this PR now.

**Review by tmszi — APPROVED** [review]

**Comment by tmszi** [comment]
> Thanks, @tmszi !

You are welcome.

**Comment by tmszi** [comment]
> The open issues as per [#290 (comment)](https://github.com/OSGeo/grass-addons/pull/290#issuecomment-699902744) might be addressed in separate PR(s).

I will try to look at it today.

> If you agree, I'd merge this PR now.

Yes I agree.

**Pull request merged**